mixing/mixclient/limits.go

```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2024 The Decred developers
```

Comment:
Adapted from https://github.com/decred/cspp/blob/master/coinjoin/size.go.

`// Copyright (c) 2023-2024 The Decred developers`